images/iscsi-demo-target-tgtd/run-tgt.sh

```diff
@@ -40,6 +46,25 @@ if [ -n "$PASSWORD" ] && [ -n "$USERNAME" ]; then
 fi
 
 LUNID=1
+
+convert_to_filesystem() {
+    local FN=$1
+    local DISK_NAME="/tmp/$(basename $FN)"
+    dd if=/dev/zero of=$DISK_NAME bs=1M count=1024
+    mkfs.ext4 $DISK_NAME
+    sleep 10
```

Comment:
@cynepco3hahue  do you know why this sleep is here?

Reply:
@davidvossel I saw that sometimes mount failed without sleep